app/react/Map/specs/__snapshots__/helper.spec.js.snap

```diff
@@ -3,11 +3,12 @@
 exports[`map helper getMarkers() Inheritance should include inherited properties 1`] = `
```

Comment:
Change the spec that is using this snapshot to test that the inherited property is in the array of markers, an snapshot here is confusing and hard to tell when the snapshot is correct

Reply:
Done, kindly check if that is what is needed.